tests/js/server/aql/aql-subquery.js

```diff
@@ -458,6 +458,48 @@ function ahuacatlSubqueryTestSuite () {
       } finally {
         db._drop(colName);
       }
+    },
+
+    testOneShardDBAndSpliceSubquery: function () {
+      const dbName = "SingleShardDB";
```

Comment:
We will probably need to add an `if` here that checks for an enterprise version build.
The one shard optimizer rule will not be triggered in community builds.

Reply:
Or move the test into the enterprise repo.